lorien/Main.gd

```diff
@@ -129,45 +129,39 @@ func _process(delta):
 # -------------------------------------------------------------------------------------------------
 func _unhandled_input(event):
 	if ! is_dialog_open():
-		var copy := Input.is_action_just_pressed("copy_strokes")
-		var paste := Input.is_action_just_pressed("paste_strokes")
-		var duplicate := Input.is_action_just_pressed("duplicate_strokes")
-		if copy || paste || duplicate:
-			return
-
-		if Input.is_action_just_pressed("toggle_player"):
```

Comment:
These are changes introduced in https://github.com/mbrlabs/Lorien/pull/168 - you can ignore them.